crates/bevy_asset/src/server/mod.rs

```diff
@@ -288,6 +288,35 @@ impl AssetServer {
         self.load_internal(None, path, false, None).await
     }
 
+    #[must_use = "not using the returned strong handle may result in the unexpected release of the assets"]
+    pub fn load_untyped<'a>(&self, path: impl Into<AssetPath<'a>>) -> Handle<LoadedUntypedAsset> {
+        let handle = {
+            let mut infos = self.data.infos.write();
+            infos.create_loading_handle::<LoadedUntypedAsset>()
+        };
+        let id = handle.id().untyped();
+        let path = path.into().into_owned();
```

Comment:
Handles to `LoadedUntypedAsset` are not shared since the paths are the same as the paths to the underlying asset. A little hack that naively seems to work fine, is to use the path with a hardcoded label. 

```suggestion
    #[must_use = "not using the returned strong handle may result in the unexpected release of the assets"]
    pub fn load_untyped<'a>(&self, path: impl Into<AssetPath<'a>>) -> Handle<LoadedUntypedAsset> {
        let path = path.into().into_owned();
        let (handle, should_load) = self
            .data
            .infos
            .write()
            .get_or_create_path_handle::<LoadedUntypedAsset>(
                path.clone().with_label("untyped"),
                HandleLoadingMode::Request,
                None,
            );
        if !should_load {
            return handle;
        }
        let id = handle.id().untyped();
```
Am I correct in thinking that these kinds of labels do not need hot-reloading, since their underlying handles should reload automatically?

Reply:
I applied the above diff, so handles to `LoadedUntypedAsset` are now shared using a label to be different form the handles to their wrapped asset.